### ISSUE TITLE
Exercise 7f

### ISSUE THREAD

**Issue by jhenrii93** [SuperSimpleDev/javascript-course#16]
Hello,  I did this exercise that way below. It is wrong? work the same way that yours. 

<html>
  <head>
    <title>Conversor de Temperatura</title>
  </head>
  <body>
    
    <script>
    
      function convertTemperature (degrees,unit){
        
        
        if (unit === 'F') {
          const result = (degrees - 32) *5 / 9
          console.log(`'${result}C'`);
        } else if (unit === 'C') {
          const result2 = (degrees *9 / 5) + 32
          console.log(`'${result2}F'`);
        }
      }

      convertTemperature (86, 'F');
      
    </script>
  </body>
</html>

**Comment by unevtable** [comment]
yeah it does work but using return makes it cleaner and minimal, and you dont have to create a variable for result.
